### PR TITLE
Update logging to process it through MDSD

### PR DESCRIPTION
**Pull request by chintanrp**
**First PR as part of migrating app-insights based telemetry pipeline to mdsd.**

- ​Change python root logger formatter to Json Formatter, which emits all log records into json format to stream/console and file
- Create request filter to add contextual info (RequestId, SessionId) of each request to log records using thread local
- Create 3 child loggers from root logger to keep track of Trace Event logging, ApiQoS logs and Metric logs for each requests.
- Format each log message as well other attributes part of message into json structure

*Following PR will contain cleanup for the app-insights references

### CODE REVIEW

File: pyServer/main.py [hunk 1]
```diff
@@ -19,9 +20,11 @@
 """
 Logger Initialization
 """
-logFormatter = logging.Formatter("%(asctime)s [%(threadName)-12.12s] [%(levelname)-7.7s]: %(message)s")
+# Use JsonFormatter to structure all log recods into json format.
```
Comment (waynekuo) [inline pyServer/main.py:23]: >recods [](http://example.com/codeflow?start=41&length=6)

Typo

File: pyServer/AzureDiskInspectService.py [hunk 2]
```diff
@@ -150,9 +180,16 @@ def InitializeAppInsights(self):
         self.rootLogger.info("AppInsights key: '" + appInsightsKey + "'")   # log locally
         self.rootLogger.info("AppInsights endpoint: '" + appInsightsEndPointUrl + "'")
 
-        # create a child logger per thread so that we can set the SessionId without collision during concurrent execution
-        #  by default logging will propagate to the parent rootLogger
-        self.telemetryLogger = self.rootLogger.getChild('AppInsights.{0}'.format(cur_thread) )
+        # TODO: AppInsights referances and comments needs to modify
```
Comment (waynekuo) [inline pyServer/AzureDiskInspectService.py:183]: >needs to modify [](http://example.com/codeflow?start=52&length=15)

need to be modified

File: pyServer/AzureDiskInspectService.py [hunk 3]
```diff
@@ -150,9 +180,16 @@ def InitializeAppInsights(self):
...
+        # create a child logger for appInsights per thread so that we can set the SessionId without collision during concurrent execution
+        # by default logging will propagate to the parent rootLogger
+        # create child loggers for trace events, ApiQos and Metrics for each requests.
```
Comment (waynekuo) [inline pyServer/AzureDiskInspectService.py:186]: >ApiQos [](http://example.com/codeflow?start=49&length=6)

ApiQo"S"

Comment (waynekuo) [inline pyServer/AzureDiskInspectService.py:186]: >trace events [](http://example.com/codeflow?start=35&length=12)

RequestEvent?

File: pyServer/AzureDiskInspectService.py (hunk 2 above)
Comment (chintanrp) [reply pyServer/AzureDiskInspectService.py]: Done

**Review by waynekuo — APPROVED** [review]
:shipit:

**Pull request merged**